### ISSUE TITLE
Blank rows in display of faunal remains by anatomical part

### ISSUE THREAD

**Issue by atomrab** [atomrab/chersark#8]
There are rows that have no value for species or part, but a count. I don't know what these represent. Is it obvious from the data in ARK what's missing here?

**Comment by atomrab** [comment]
These are also present in the text export that we used to go from the database to ARK. I can't explain it without opening the original mdb file, which I'll try to do in the immediate future.

**Comment by atomrab** [comment]
Action: Adam will check cxt CH06SR_712 against previous versions of the faunal database to see if the blank rows were there since the earliest version. If the entity the count refers to can't be reconstructed, rows with no value for "species" will be deleted.

**Comment by MikeJJohnson** [comment]
Did you find anything on this? Am I ok to go ahead and remove these junk numbers?

**Comment by atomrab** [comment]
Ok, I have figured this all out. The numbers are not junk, although there are some blank rows in the actual database, which seem already to have been purged in the tables (for example, there are nine blank rows for CH06SR_884, but these don't show up in either the species or the part tables). 

The blank rows represent species that somehow weren't mapped to the ARK database and/or were dropped during export somehow. They're all fish and molluscs; we may have decided that we'd leave these to the fish specialists or something, but I think it's worth including them, especially because the shell counts don't appear anywhere else. 

These species are the following: "Sturgeon", "Helix", "Cardium cost", "Catfish", "Pickerel", "Pike", "Rutilus frisii", and "Spirillus". I've mapped all of these to NCBI and EOL, erring on the side of generality (so the Cardium shells are just under Cardiidae, the pike and pickerel just under Esox, and the sturgeon under Aceripensidae). The only questionable one is Spirillus, which isn't a species name that I could find anywhere at all; since they're referred to in terms of numbers of shells (and not valves) in the report, my best guess is that they're marine gastropods, so I've mapped them to Neogastropoda (whelks and cone-shells).

I'm uploading the latest version of the species mapping document, if we can get in and fix those entries, and I will email the CSV export of the most recent version of the faunal database that I have to @MikeJJohnson separately. I've also identified another problem with faunal counts that I will open a new issue for.
